vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -10,6 +10,9 @@
 #include "Impl/ScrollViewUWPImplementation.h"
 #include "ScrollViewManager.h"
 
+#include <winrt/Windows.UI.Core.h>
+#include <winrt/Windows.UI.Xaml.Input.h>
```

Comment:
these should be using the straddle headers, not the cppwinrt ones
(#include "UI.Core.h" etc.)

Reply:
What are straddle headers and why should I use them here?

Reply:
the headers that allow us to reason about a xaml type regardless of whether it's in UWP XAML or WinUI 3.
You should use them here so that you don't break WinUI 3 RNW builds.